### DISCUSSION TITLE
TypeError: spacy.load is not a function

### DISCUSSION THREAD

**Discussion by steiner-0** [explosion/spaCy#11106] [Help: Coding & Implementations] [answered]
I was testing the example code on the website, but it gives an error that spacy.load is not a function. I checked the manual and it says spacy.load is a module. I am confused. Where can I import that module?
![Screen Shot 2022-07-10 at 3 28 37 AM](https://user-images.githubusercontent.com/71510082/178120126-16e5501b-5d13-4a9e-8b10-83a0e99ecaa3.png)

**Comment by polm** [comment, marked as answer]
It looks like your sample code is from [spacy-js](https://spacy.io/universe/project/spacy-js) in the Universe? In order for that to work you need to install both spacy and that library; it looks like you didn't install `spacy-js`. However, since that hasn't been updated in a while, I'm not sure it'll work with spaCy v3.
